yt/frontends/enzo/misc.py

```diff
@@ -13,3 +13,29 @@
 # The full license is in the file COPYING.txt, distributed with this software.
 #-----------------------------------------------------------------------------
 
+import numpy as np
+
+from yt.utilities.physical_ratios import \
+    rho_crit_g_cm3_h2, cm_per_mpc
+
+def cosmology_get_units(hubble_constant, omega_matter, box_size,
+                        initial_redshift, current_redshift):
+    """
+    Return a dict of Enzo cosmological unit conversions.
+    """
+
+    zp1  = 1.0 + current_redshift
+    zip1 = 1.0 + initial_redshift
+
+    k = {}
+    k["utim"] = 2.52e17 / np.sqrt(omega_matter) / \
```

Comment:
Are these constants hardcoded within Enzo? If so, it might be worth having a comment there.

Reply:
Good idea.